composer.json

```diff
@@ -37,20 +37,21 @@
         }
     },
     "require": {
-        "php": ">=7.4",
+        "php": ">=8.2",
         "ext-curl": "*",
         "ext-json": "*",
         "ext-zip": "*",
         "drupal/search_api_solr": "^4.3",
-        "guzzlehttp/guzzle": "^6.5.7|^7.5",
-        "http-interop/http-factory-guzzle": "^1.0",
-        "kint-php/kint": "^4.1|^5",
+        "guzzlehttp/guzzle": "^7.5",
+        "guzzlehttp/psr7": "^2.7",
+        "kint-php/kint": "^5|^6",
+        "php-http/guzzle7-adapter": "^1.1",
         "psr/event-dispatcher": "^1.0",
-        "symfony/finder": "^4.4|^5|^6",
-        "symfony/yaml": "^4.4|^5|^6"
+        "symfony/finder": "^4.4|^5|^6|^7",
+        "symfony/yaml": "^4.4|^5|^6|^7"
     },
     "require-dev": {
-        "consolidation/robo": "^3.0",
+        "consolidation/robo": "^4",
```

Comment:
This will constrain you to PHP 8.2. If you want to be able to test on 8.1 through 8.4, using `^3 || ^4` should work, as long as you are judicious about running `composer update` at the right time in CI